addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -470,11 +469,11 @@
             <div class="d-flex align-items-center small">
                 <span class="font-weight-bold mr-auto" t-field="slide.completion_time" t-options='{"widget": "float_time"}'/>
                 <div class="o_wslides_js_slide_like mr-2">
-                    <span t-att-class="('o_wslides_js_slide_like_up %s') % ('disabled' if not (slide_access['can_access'] and slide_access['can_vote']) else '')" tabindex="0" data-toggle="popover" t-att-data-slide-id="slide.id">
+                    <span t-att-class="('o_wslides_js_slide_like_up %s') % ('disabled' if not channel.can_vote else '')" tabindex="0" data-toggle="popover" t-att-data-slide-id="slide.id">
```

Comment:
Parenthesis around string are not necessary I think (t-att-class=" 'my_string %s' % ()") - same just below